src/execution_plan/ops/shared/update_labels.c

```
@@ -0,0 +1,131 @@
+		GrB_Vector v  = NULL  ;
+		bool resolved = false ;
+
+		// determine label ID
+		Schema *s = GraphContext_GetSchema (gc, lbls[i], SCHEMA_NODE) ;
+		label_ids [i] = (s != NULL) ? Schema_GetID (s) : GRAPH_UNKNOWN_LABEL ;
+
+		for (uint j = 0 ; j < *n_vecs ; j++) {
+			v = _vecs [j] ;
+			GrB_OK (GrB_get (v, name, GrB_NAME)) ;
+			if (strcmp (lbls[i], name) == 0) {
+				rm_free (name) ;
```

Cppcheck (2.20.0)

[error] Deallocation of an auto-variable results in undefined behaviour.

(autovarInvalidDeallocation)